src/jira/util/queryAtlassianConnectPublicKey.ts

```diff
@@ -0,0 +1,23 @@
+import axios from "axios";
+
+const CONNECT_INSTALL_KEYS_CDN_URL =
+	"https://connect-install-keys.atlassian.com";
```

Comment:
We need to be aware of one thing. This is a link for pord. After this change we won't be able to use the app with `jira-dev` 

And why would you need hence we can easily can use prod jira for the app local testing? But I am letting you know in case if anyone needs it